Sources/TuistSupportTesting/Extensions/XCTestCase+Extras.swift

```diff
@@ -140,6 +140,20 @@ extension XCTestCase {
         XCTFail("No error was thrown", file: file, line: line)
     }
 
+    public func XCTAssertThrows(
+        _ closure: @autoclosure () async throws -> some Any,
+        file: StaticString = #file,
+        line: UInt = #line
+    ) async {
+        do {
+            _ = try await closure()
+        } catch {
+            XCTAssertTrue(true)
+            return
+        }
+        XCTFail("No error was thrown", file: file, line: line)
```

Comment:
```suggestion
        do {
            _ = try await closure()
            XCTFail("No error was thrown", file: file, line: line)
        } catch {
            // Succeeded
        }
        
```

Wouldn't this do the same? Is there a good reason for `XCTAssertTrue(true)`?

Reply:
No strong reason, yours reads nicely 👍